packages/now-cli/src/util/dev/builder-cache.ts

```diff
@@ -166,12 +99,20 @@ function parseVersionSafe(rawSpec: string) {
 export function filterPackage(
   builderSpec: string,
   distTag: string,
-  buildersPkg: PackageJson
+  buildersPkg: PackageJson,
+  nowCliPkg: PackageJson = {}
```

Comment:
I think this is a bug. This local `nowCliPkg` is going to mask the global `nowCliPkg`, right?

Reply:
The global one gets passed into this function for all usages. This parameter is so that the current unit tests for this function pass without any changes (as essentially the tests don't have any bundled builders cause of `{}`).